src/generic-container.ts

```diff
@@ -35,6 +35,9 @@ import { HostPortWaitStrategy, WaitStrategy } from "./wait-strategy";
 
 export class GenericContainerBuilder {
   private buildArgs: BuildArgs = {};
+  private imageName: string | null = null;
```

Comment:
Can we not use optional syntax here? `imageName?`

Reply:
Sure. I just forgot about it. I'll now stucked in meetings etc. but hopefully can take care of it later that day ;)

Reply:
done